spock-core/src/main/java/spock/lang/Rollup.java

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright 2020 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package spock.lang;
+
+import org.spockframework.util.Beta;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+/**
+ * Indicates that iterations of a data-driven feature should not be made visible
+ * as separate features to the outside world (IDEs, reports, etc.) but as one atomic test.
+ * The {@code Rollup} annotation can also be put on a spec class. This has the same
```

Comment:
Den Satz bzgl. Spec sollte bei `@Unroll` auch ran
Und IMHO sollte auch was zur inheritance gesagt werden.

Reply:
`@Unroll` JavaDoc is where I copied that from.
It is basically the same JavaDoc, just with two or three words adapted and the part about naming removed that does not apply to uprolling.

Regarding inheritance, I thought not having the annotation annotated with `@Inherited` should be enough documentation. But I added a sentence to both annotations.